fair/tools/unifiedtools.py

```diff
@@ -196,6 +199,86 @@ def create_output_dataframes(inp_df, gas_np, RF_np, T_np, alpha_np, ext_forcing_
     )
     return gas_df, RF_df, T_df, alpha_df
 
+def create_output_dataframe_aimc_compliant(inp_df, gas_np, RF_np, T_np, alpha_np, ext_forcing_np):
+    """
+    TODO: docstring
+    """
+
+    units = Units()
+
+    variable_array = inp_df.timeseries().index.levels[3].to_numpy()
```

Comment:
```suggestion
    inp_df_ts = inp_df.timeseries()
    variable_index_level = inp_df_ts.index.names.tolist().index("variable")
    variable_array = inp_df_ts.index.levels[variable_index_level].to_numpy()
```

I think something like that is safer and easier to read rather than having this magic number 3 just appear

Reply:
Yes, good point, I've tried to avoid this elsewhere